src/skk.cpp

```diff
@@ -517,19 +523,18 @@ void SkkEngine::loadDictionary() {
                     }
                 }
             } else {
-                constexpr char configDir[] = "$FCITX_CONFIG_DIR/";
-                constexpr auto len = sizeof(configDir) - 1;
-                std::string realpath = path;
-                if (stringutils::startsWith(path, configDir)) {
-                    realpath = stringutils::joinPath(
+                std::string_view partialpath = path;
+                if (stringutils::consumePrefix(partialpath,
+                                               "$FCITX_CONFIG_DIR/")) {
+                    path = stringutils::joinPath(
                         StandardPath::global().userDirectory(
                             StandardPath::Type::PkgData),
-                        path.substr(len));
+                        std::string(partialpath));
```

Comment:
I believe you don't need to cast to std::string here

Reply:
Indeed it's different from `locate`.